src/yang/ietf-traffic-policy.yang

```diff
@@ -227,7 +227,10 @@ module ietf-traffic-policy {
       }
 
       leaf name {
-        type string;
+        type leafref  {
+          path "/traffic-policy:policies/" +
+              "traffic-policy:policy/traffic-policy:name";
+        }
```

Comment:
You say in your comments related to this issue that:

_Policy refers classifier. Classifier is defined inline or as template. Template can be referred. So, policy can use leafref if classifier type can be defined as union of local name and leafref of classifier template._

For the last sentence, I have not looked at the classifier definition. Is the reference to it defined as a union of local name (for inline) and a leafref to template?